source/includes/_webhooks.md

```diff
@@ -581,17 +581,13 @@ This event is fired two times in the lifecycle of a dividend post request.
         "currency": "GBP",
         "amount": 100
       },
-      "payments": [
-        {
-          "clientId": "clientId",
-          "accountType": "GIA",
-          "payment": {
-            "currency": "GBP",
-            "amount": 100
-          },
-          "status": "DISTRIBUTED"
-        }
-      ],
+      "paymentsSummary": {
```

Comment:
this is more to do with the dto field name... i reckon changing 'paymentsSummary' to just 'summary' makes sense (in your other PR), given the 'type' 'DIVIDEND_UPDATE' is known && the fields within the object e.g. 'totalPayments' makes it obvious that the overall obj. is a summary of payments.